django_comments_xtd/api/frontend.py

```diff
@@ -143,6 +143,10 @@ def get_props(cls, obj, user, request=None):
             d['like_url'] = reverse("comments-xtd-like", args=(0,))
             d['dislike_url'] = reverse("comments-xtd-dislike", args=(0,))
 
+        defaults_per_model = settings.COMMENTS_XTD_APP_MODEL_OPTIONS[str(ctype).replace(' | ', '.')]
```

Comment:
I think that on line 92 above, the options for the app model might have already been collected (`options = get_app_model_options(content_type=ctype_key`) so down here it could be sufficient to `d.update(options)`.

See also l.100 `"who_can_post": options['who_can_post'],`.

However, I'm not sure if, at the moment, the function `get_app_model_options()` does the `replace(' | ', '.')` trick